doc/man1/openssl-rehash.pod

```diff
@@ -11,12 +11,12 @@ openssl-c_rehash - Create symbolic links to files named by the hash values
 
 B<openssl>
 B<rehash>
-B<[-h]>
-B<[-help]>
-B<[-old]>
-B<[-n]>
-B<[-v]>
-[ I<directory>...]
+B<-h>
```

Comment:
Why are the  brackets removed here?

Reply:
Because I goofed, it should be ```[B<-goo>]```.  good catch.  fixed.